api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/ContentModelSetup.java

```diff
@@ -191,6 +213,144 @@ private void renamePortal(Resource portal, String namespace, Model model) {
+        OntModel applicationMetadataModel = models.getOntModel(APPLICATION_METADATA_FIRSTTIME);
+        OntModel baseABoxModel = models.getOntModel(ABOX_ASSERTIONS_FIRSTTIME);
+        OntModel baseTBoxModel = models.getOntModel(TBOX_ASSERTIONS_FIRSTTIME);
+
+
+        // check if ApplicationMetadataModel is the same in file and configuration models
+        log.debug("compare firsttime files with configuration models (backup from first start) for Application Metadata Model");
+        OntModel testApplicationMetadataModel = VitroModelFactory.createOntologyModel();
+        RDFFilesLoader.loadFirstTimeFiles(ctx, "applicationMetadata", testApplicationMetadataModel, true);
+        setPortalUriOnFirstTime(testApplicationMetadataModel, ctx);
+    
+        if ( applicationMetadataModel.isIsomorphicWith(testApplicationMetadataModel) ) {
+            log.debug("They are the same, do nothing");
+        } else {
+            log.debug("they differ, compare values in configuration models with user's triplestore");     
+            OntModel userTriplestoreApplicationMetadataModel = models.getOntModel(APPLICATION_METADATA);
+
+            // double check the statements (blank notes, etc.) and apply the changes
+            updatedFiles = applyChanges(applicationMetadataModel, testApplicationMetadataModel, userTriplestoreApplicationMetadataModel, "applicationMetadata");
+            if (updatedFiles) log.info("The applicationMetadata model was updated.");
+        }
+
+        // check if abox model is the same in file and configuration models
+        log.debug("compare firsttime files with configuration models (backup from first start) for Base ABox Model");
+        OntModel testBaseABoxModel = VitroModelFactory.createOntologyModel();
+        RDFFilesLoader.loadFirstTimeFiles(ctx, "abox", testBaseABoxModel, true);
+
+        if ( baseABoxModel.isIsomorphicWith(testBaseABoxModel) ) {
+            log.debug("They are the same, so do nothing");
+        } else {
+            log.debug("they differ, compare values in configuration models with user's triplestore");
+            OntModel userTriplestoreABoxModel = models.getOntModel(ABOX_ASSERTIONS);
+
+            // double check the statements (blank notes, etc.) and apply the changes
+            updatedFiles = applyChanges(baseABoxModel, testBaseABoxModel, userTriplestoreABoxModel, "abox");
+            if (updatedFiles) log.info("The ABox model was updated.");
+        }
+
+
+        // check if tbox model is the same in file and configuration models
+        log.debug("compare firsttime files with configuration models (backup from first start) for Base TBox Model");
+        OntModel testBaseTBoxModel = VitroModelFactory.createOntologyModel();
+        RDFFilesLoader.loadFirstTimeFiles(ctx, "tbox", testBaseTBoxModel, true);
+
+        if ( baseTBoxModel.isIsomorphicWith(testBaseTBoxModel) ) {
+            log.debug("They are the same, so do nothing");
```

Comment:
Could you please change the logging message as follows:
```
log.debug("They are the same, so do nothing: '" + modelPath + "'");
```

Reply:
i changed it